test/prefix_test.rb

```diff
@@ -95,7 +95,7 @@ def call(env)
 
     %w(GET POST PUT PATCH DELETE TRACE OPTIONS).each do |verb|
       it "recognizes requests (#{ verb })" do
-        env = Rack::MockRequest.env_for('/admin/home', method: verb)
+        env = Rack::MockRequest.env_for('/home', method: verb)
```

Comment:
ok, with the problem, now path_prefix is only for path and url generation, if we use like namespace we have problems with lotus architecture

Then we can use namespace if we want that namespace in the request url and path_prefix only for path and url helpers